js_modules/dagit/packages/core/src/asset-graph/AssetNode.tsx

```diff
@@ -78,7 +78,9 @@ export const AssetNode: React.FC<{
           {runOrError?.__typename === 'Run' && event ? (
             <>
               <StatsRow>
-                {runOrError.pipelineName !== __ASSET_GROUP ? (
+                {runOrError.pipelineName.startsWith(__ASSET_GROUP_PREFIX) ? (
```

Comment:
Would it be worth extracting out a utility function that does the prefix check? That way the prefix const and `startsWith` checks don't have to be everywhere, and we'd futureproof for a bit more flexibility later if we introduced other valid prefixes or something.